### PR TITLE
cmd/tailscale/cli: update serve/funnel CLI help text

### PR DESCRIPTION
**Pull request by tylersmalley**
updates #8489
ENG-2308

### CODE REVIEW

File: cmd/tailscale/cli/serve_v2.go [hunk 1]
```diff
@@ -39,15 +39,22 @@ type commandInfo struct {
...
-<target> can be a port number (e.g., 3000), a partial URL (e.g., localhost:3000), or a
-full URL including a path (e.g., http://localhost:3000/foo, https+insecure://localhost:3000/foo).
+<target> can be a file, directory, text, or most commonly the location to a service running on the
+local machine. The location to the location service can be expressed as a port number (e.g., 3000),
+a partial URL (e.g., localhost:3000), or a full URL including a path (e.g., http://localhost:3000/foo).
 
 EXAMPLES
-  - Mount a local web server at 127.0.0.1:3000 in the foreground:
-    $ tailscale %s localhost:3000
+  - Expose an HTTP serve running at 127.0.0.1:3000 in the foreground:
```
Comment (tylersmalley) [inline cmd/tailscale/cli/serve_v2.go:47]: @samlinville thoughts on the change of wording away from using "mount"?

Comment (samlinville) [reply cmd/tailscale/cli/serve_v2.go:47]: Fine with it — what's your suggestion for an alternative?

Comment (tylersmalley) [reply cmd/tailscale/cli/serve_v2.go:47]: It's in the PR, mostly using the wording from the CLI flags.

Comment (samlinville) [reply cmd/tailscale/cli/serve_v2.go:47]: SGTM

Comment (samlinville) [inline cmd/tailscale/cli/serve_v2.go:47]:
```suggestion
  - Expose an HTTP server running at 127.0.0.1:3000 in the foreground:
```

**Review by marwan-at-work — APPROVED** [review]

**Pull request merged**